docs/installation/setup-gce.rst

```diff
@@ -11,7 +11,7 @@ You'll need to setup at least two nodes.
 
 #. Create a new cloud server:
 
-   * Visit the `Google Cloud Console <https://console.cloud.google.com/>`_.
+   * Visit the `Google Cloud Console <https://console.cloud.google.com/start>`_.
```

Comment:
Okay... so Google 404's if you aren't logged in, that's annoying. https://cloud.google.com/compute/docs/console might be an okay alternative.